### ISSUE TITLE
[Quest][Classic] Tiragarde Keep 

### ISSUE THREAD

**Issue by Solpowered** [dalaranwow/dalaran-wow#4784]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
After killing Benedict for the quest, you get his key to open the chest on the roof. the action of opening of the chest works fine, but when you actually open the chest, there is no envelope (item that starts another quest)
**Expected behaviour**: Tell us what should happen instead
The envelope should be in the chest. 
**Steps to reproduce the problem**:

1. Kill Benedict/ loot key.
2. Proceed to roof and the open chest.
3. Loot view shows up without the envelope being there

**Include proofs for this behaviour**
http://wow.gamepedia.com/Lieutenant_Benedict
http://wow.gamepedia.com/Benedict%27s_Chest
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
http://wow.gamepedia.com/Lieutenant_Benedict
http://wow.gamepedia.com/Benedict%27s_Chest
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
